src/redux/modules/playlist.js

```diff
@@ -1,93 +1,27 @@
+import { actions as playerActions } from './player';
+
 import { DEFAULT_CONTENT_LANGUAGE, VS_DEFAULT } from '../../helpers/consts';
-import { types as playerTypes } from './player';
 import { saveTimeOnLocalstorage } from '../../components/Player/Controls/helper';
 import { getQualitiesFromLS } from '../../pkg/jwpAdapter/adapter';
 
 export const SHOWED_PLAYLIST_ITEMS = 7;
 
-const PLAYLIST_BUILD         = 'Playlist/BUILD';
-const PLAYLIST_BUILD_SUCCESS = 'Playlist/BUILD_SUCCESS';
-const SINGLE_MEDIA_BUILD     = 'Playlist/SINGLE_MEDIA_BUILD';
-const MY_PLAYLIST_BUILD      = 'Playlist/MY_PLAYLIST_BUILD';
-
-const PLAYLIST_SELECT          = 'Playlist/SELECT';
-const PLAYER_SET_QUALITY       = 'Player/SET_QUALITY';
-const PLAYER_SET_LANGUAGE      = 'Player/SET_LANGUAGE';
-const PLAYER_SET_MEDIA_TYPE    = 'Player/SET_MEDIA_TYPE';
-const PLAYER_SET_SUBS_LANGUAGE = 'Player/SET_SUBS_LANGUAGE';
-const PLAYER_NULL_NEXT_UNIT    = 'Player/NULL_NEXT_UNIT';
-const SHOW_IMAGES              = 'Player/SHOW_IMAGES';
-const FETCH_SHOW_DATA          = 'Player/FETCH_SHOW_DATA';
-const FETCH_SHOW_DATA_SUCCESS  = 'Player/FETCH_SHOW_DATA_SUCCESS';
-
-export const types = {
-  PLAYLIST_BUILD,
-  SINGLE_MEDIA_BUILD,
-  MY_PLAYLIST_BUILD,
-  FETCH_SHOW_DATA
-};
-
-// Actions
-const build            = createAction(PLAYLIST_BUILD, (cId, cuId) => ({ cId, cuId }));
-const buildSuccess     = createAction(PLAYLIST_BUILD_SUCCESS);
-const singleMediaBuild = createAction(SINGLE_MEDIA_BUILD);
-const myPlaylistBuild  = createAction(MY_PLAYLIST_BUILD, pId => ({ pId }));
-
-const select               = createAction(PLAYLIST_SELECT);
-const setQuality           = createAction(PLAYER_SET_QUALITY);
-const setLanguage          = createAction(PLAYER_SET_LANGUAGE);
-const setSubsLanguage      = createAction(PLAYER_SET_SUBS_LANGUAGE);
-const setMediaType         = createAction(PLAYER_SET_MEDIA_TYPE);
-const nullNextUnit         = createAction(PLAYER_NULL_NEXT_UNIT);
-const showImages           = createAction(SHOW_IMAGES);
-const fetchShowData        = createAction(FETCH_SHOW_DATA);
-const fetchShowDataSuccess = createAction(FETCH_SHOW_DATA_SUCCESS);
-
-export const actions = {
-  build,
-  singleMediaBuild,
-  myPlaylistBuild,
-  buildSuccess,
-
-  select,
-  setQuality,
-  setLanguage,
-  setSubsLanguage,
-  setMediaType,
-  nullNextUnit,
-  showImages,
-  fetchShowData,
-  fetchShowDataSuccess
-};
-
-/* Reducer */
-const initialState = {
-  playlist: [],
-  itemById: {},
-  info: {},
-  fetched: {}
-};
-
-const onBuild = draft => {
-  draft.info = { isReady: false, wip: true };
-};
-
-const onBuildSuccess = (draft, payload) => {
+const onBuildSuccess = (status, payload) => {
   const { cuId, id: _id, items, fetched = {}, ...info } = payload;
 
   const id     = _id || cuId;
-  let language = draft.info.language || payload.language || DEFAULT_CONTENT_LANGUAGE;
+  let language = status.info.language || payload.language || DEFAULT_CONTENT_LANGUAGE;
```

Comment:
you mean state instead status?

Reply:
yep, good catch
(though this callback got it as a parameter, so this is just proper naming case, but yes, fixed in this an other functions)